tests/system/action/meeting/test_clone.py

```diff
@@ -1285,7 +1285,7 @@ def test_meeting_name_too_long(self) -> None:
         self.assert_status_code(response, 200)
         self.assert_model_exists("meeting/2", {"name": "A" * 90 + "... - Copy"})
 
-    def test_permissions_both_okay(self) -> None:
+    def test_permissions_explicit_source_committee_permission(self) -> None:
         self.set_models(self.test_models)
         self.set_models(
             {
```

Comment:
Committee 2 is part of the test models in this class, these are used in a lot of different tests. Removing the committee would therefore have to be done locally in this test to avoid impacting all the others, which would just mean extra test length with no actual benefit for efficiency

Reply:
I can only see committee 1 in the test models, which is the one you are using.